tessera-core/src/main/java/com/quorum/tessera/sync/TransactionRequesterImpl.java

```diff
@@ -72,10 +74,11 @@ private boolean makeRequest(final String uri, final ResendRequest request) {
      * @param key the public key that transactions should be resent for
      * @return the request to be sent
      */
-    private ResendRequest createRequestAllEntity(final Key key) {
+    private ResendRequest createRequestAllEntity(final PublicKey key) {
 
         final ResendRequest request = new ResendRequest();
-        request.setPublicKey(key.toString());
+        String encoded = Base64.getEncoder().encodeToString(key.getKeyBytes());
+        request.setPublicKey(encoded);
```

Comment:
Should use the Key method to encode it to string instead of getting an encoder on the bytes